config.yaml.dist

```diff
@@ -31,6 +31,16 @@ clusterNetwork:
   serviceDomainName: ""
   # a nodePort range to reserve for services (default: 30000-32767)
   nodePortRange: ""
+  # CNI plugin choice
```

Comment:
Please update the manifest used for the `config print` command.

Reply:
OK, we need to get rid of `config.yaml.dist` completely as it doesn't make sense anymore in the presence of `config print`